packages/e2e/src/load-test-scheduler.ts

```diff
@@ -0,0 +1,30 @@
+import { Logger } from 'ts-log';
+import { Observable, map, mergeMap, take, timer } from 'rxjs';
+
+export interface LoadTestSchedulerProps<T> {
+  duration: number;
+  callsPerDuration: number;
+  idOffset?: number;
+  callUnderTest: (id: number) => Observable<T>;
+}
+
+export interface LoadTestSchedulerDependencies {
+  logger: Logger;
+}
+
```

Comment:
Doc comments are missing

Reply:
Updated in [807d2a8](https://github.com/input-output-hk/cardano-js-sdk/pull/526/commits/807d2a86f40f263f4f62b39e3a61ac5797fd3da7)